#src/scss/blocks/nav.scss

```diff
@@ -0,0 +1,21 @@
+.nav__list {
```

Comment:
all `ul` or other list tags have default padding besides list-style-type, so make sure this pre-set default padding doesn't affect your list
Usually, I immediately set padding on all list elements to 0, if don't need them